src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -124,6 +124,11 @@ void fillInnerParts() {
         commandBoxPlaceholder.getChildren().add(commandBox.getRoot());
     }
 
+    void clearTaskListPanel() {
```

Comment:
One small nit, perhaps we could make this `private` instead?

Reply:
Sure!